### PR TITLE
Turkish Translation

### PR DESCRIPTION
**Pull request by ibrakap**
This pull request adds Turkish translation.

### CODE REVIEW

File: po/tr.po [hunk 1]
```diff
@@ -0,0 +1,531 @@
...
+msgid "Artist"
+msgstr "Şarkıcı"
+
+#: src/Views/Albums.vala:78 src/Dialogs/TrackEditor.vala:90
+msgid "Year"
+msgstr "Yıl"
+
+#: src/Views/Albums.vala:79 src/Dialogs/TrackEditor.vala:88
+msgid "Genre"
+msgstr Tür
```
Comment (ryonakano) [inline po/tr.po]:
```suggestion
msgstr "Tür"
```

Comment (ibrakap) [reply po/tr.po]: Ah my mistakes. I will fix it.

**Comment by alainm23** [comment]
Thank you very much for the translation.

**Comment by ryonakano** [comment]
@ibrakap If you add `po/extra/tr.po` you also need to add `tr` to the `po/extra/LINGUAS` so that the Turkish translation file can be updated with other translation files.

**Pull request merged**